demos/python_demos/image_inpainting_demo/image_inpainting_demo.py

```diff
@@ -82,24 +81,25 @@ def create_random_mask(parts, max_vertex, max_length, max_brush_width, h, w, max
     return mask
 
 
-def inpaint_random_holes(img, args):
-    mask_color = args.mask_color[::-1] # argument comes in RGB mode, but we will use BGR notation below
-
+def inpaint_auto(img, args):
     ie = IECore()
 
     inpainting_processor = ImageInpainting(ie, args.model, args.device)
 
     #--- Resize to model input and generate random mask
```

Comment:
This comment is not quite correct: 1) the order of operations is now reversed; 2) the mask is not necessarily random.

Reply:
Corrected